### PR TITLE
Add `bento` to the list of Deployment tools

### PR DESCRIPTION
**Pull request by anoadragon453**
Adds [bento](https://github.com/rapenne-s/bento/) - yet another tool to deploy your nix systems.

cc @rapenne-s

### CODE REVIEW

**Review by Mic92 — APPROVED** [review]

**Review by rapenne-s — APPROVED** [review]
thanks! looks good to me

File: README.md [hunk 1]
```diff
@@ -73,6 +73,7 @@
 
 ## Deployment Tools
 
+* [bento](https://github.com/rapenne-s/bento/) - A KISS deployment tool to keep your NixOS fleet (servers & workstations) up to date.
```
Comment (houstdav000) [inline README.md:76]: The tool itself looks pretty cool and continually developed, so its probably good to add.

As a nitpick, trim the last slash from the URL please.

More seriously, the given description doesn't do a great job of making it distinct from the other deployment tools. I'd recommend mentioning it's pull-based and that it leverages SSH.

Perhaps something like "A pull-based deployment agent to keep your NixOS fleet up-to-date over SSH."?

**Review by houstdav000 — CHANGES_REQUESTED** [review]
Looks good, but could be better. Check the comment for info

**Comment by houstdav000** [comment]
I'll take a look when I get home about that API limit GitHub Action failure

**Comment by Mic92** [comment]
> I'll take a look when I get home about that API limit GitHub Action failure

Should fixed now?

**Comment by houstdav000** [comment]
Should be. Both the number of requests should have been reduced with the version upgrade, and the token should lift our request limit. I'll verify in a second though

**Review by Mic92 — APPROVED** [review]

**Comment by houstdav000** [comment]
I'd still personally like the description clarified as with that review I left, but I suppose it may work as is

**Review by houstdav000 — APPROVED** [review]
Thought I made that a review. Here we go

**Pull request merged**